src/components/cards/BiobankReportCard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="mg-biobank-card container">
+    <script v-html='jsonld' type='application/ld+json' />
```

Comment:
better use v-text (security), could you make the property more specific? e.g. collectionBioschema / biobankBioschema